src/main/java/io/openshift/launchpad/ui/booster/ProjectInfoStep.java

```diff
@@ -91,9 +99,9 @@
    public void initializeUI(UIBuilder builder) throws Exception
    {
       UIContext context = builder.getUIContext();
+      Mission mission = (Mission) context.getAttributeMap().get(Mission.class);
```

Comment:
Aha, ok, so if I want to use those values outside of that lambda I need to repeat those lines, but make sure they occur *after* the `.setDefaultValue()`?